### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by michaelrabie**
Add documentation on using spiderwasp locally and running tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## Spider Wasp JR
+## SpiderWasp JR
 
-Spiderwasp is a great microservice, but because it's in Java, it increases the barrier to extending and properly testing it with tools we are familiar with, such as RSpec.
+SpiderWasp is a great microservice, but because it's in Java, it increases the barrier to extending and properly testing it with tools we are familiar with, such as RSpec.
```
Comment (daemonsy) [inline README.md]: Let's use the repository name as the name. No one will know the references to the old spiderwasp too

File: README.md [hunk 2]
```diff
@@ -9,3 +9,36 @@ This is an experimental rewrite of spiderwasp using:
 - RSpec
 - Trinidad for serving production requests
```
Comment (daemonsy) [inline README.md]: No more Trinidad, just puma now

File: README.md [hunk 3]
```diff
@@ -9,3 +9,36 @@ This is an experimental rewrite of spiderwasp using:
...
 - Puma for development (for working with pry)
+
+## Using SpiderWasp
+
+To run SpiderWasp locally:
```
Comment (daemonsy) [inline README.md]: Bonus points for adding graphiql

Comment (michaelrabie) [inline README.md]: Separate PR :p

File: README.md [hunk 4]
```diff
@@ -1,11 +1,43 @@
...
 - JRuby to provide bindings to `libphonenumber`
 - Sinatra (we can change it to anythin else, just something simple to start off)
 - RSpec
-- Trinidad for serving production requests
 - Puma for development (for working with pry)
+
+## Using PhoneLookup
+
+To run PhoneLookup locally:
+1. Run the server with the following command: `ruby app.rb`. Make a note of the port the server is running on (example localhost:4567)
```
Comment (daemonsy) [inline README.md]: For the boot command, I think we can use the same one as the [Procfile](https://github.com/alphasights/phone_lookup/blob/master/Procfile#L1)

File: README.md [hunk 5]
```diff
@@ -1,11 +1,43 @@
...
+2. Use a REST Client like [Postman](https://www.getpostman.com/) to hit the PhoneLookup API.
+3. Make a POST request in the REST Client to the graph endpoint `localhost:4567/graph` or to the respective port PhoneLookup is running on.
```
Comment (daemonsy) [inline README.md]: I think the default port in `puma.rb` is set to `3000`. But let's change the value in `puma.rb` to `4567` so not to compete with Rails

File: README.md [hunk 6]
```diff
@@ -1,11 +1,43 @@
...
+
+An example of a POST request body:
+```
+{
+    "query": "query($text: String!, $region: String) { parse_phone_number(number: $text, region: $region) {e164_format valid country_code national_number}}",
+    "variables": { "text": "+19179032036", "region": "US" }
```
Comment (daemonsy) [inline README.md]: my phone number :(

Comment (michaelrabie) [inline README.md]: I'll change that...

**Pull request merged**